sys-apps/likwid/likwid-4.1.2.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+FORTRAN_NEEDED=fortran
+
+inherit fcaps linux-info fortran-2
+
+DESCRIPTION="A performance-oriented tool suite for x86 multicore environments"
+HOMEPAGE="https://github.com/rrze-likwid/likwid"
+SRC_URI="https://ftp.fau.de/pub/likwid/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="fortran"
+
+RDEPEND="dev-lang/perl"
+
+DEPEND="${RDEPEND}
+	fortran? ( sys-devel/gcc:*[fortran] )
+	dev-lang/lua:0"
+
+CONFIG_CHECK="~X86_MSR"
+
+FILECAPS=(
+	-M 755 cap_sys_rawio usr/sbin/likwid-accessD --
+	-M 755 cap_sys_rawio usr/bin/likwid-{perfctr,bench,powermeter}
+)
+
+# See Bug 558402
+PATCHES=(
+	"${FILESDIR}/${P}-Makefile.patch"
+	"${FILESDIR}/${P}-fix-gnustack.patch"
+	"${FILESDIR}/${P}-config.mk.patch"
```

Comment:
I know there is no patches for 4.1.2 on the above link. Since I have not added them to this PR.
As you requested.

When the ebuild is committed the patches will need to be renamed, the contents of the patches apply fine.